[{"ContainerToolsException.Message":"Docker command failed with exit code 1. #1 [internal] load build definition from Dockerfile #1 sha256:9313b2f44313230c943697ab74e683401541514febcc03e36b4553bae24241de #1 transferring dockerfile: 32B 0.0s done #1 DONE 0.0s  #2 [internal] load .dockerignore #2 sha256:037160032069b1e7222fef32a9e08ed96a2b15a56ff8e18c59ced39e47170e48 #2 transferring context: 35B done #2 DONE 0.0s  #3 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0 #3 sha256:9eb4f6c3944cfcbfe18b9f1a753c769fc35341309a8d4a21f8937f47e94c712b #3 DONE 0.0s  #4 [internal] load metadata for mcr.microsoft.com/dotnet/aspnet:6.0 #4 sha256:ac4494cbca04ddb415c76edcbcc7688784c2a6ea65dd656286c013738aa3b75f #4 DONE 0.0s  #5 [base 1/2] FROM mcr.microsoft.com/dotnet/aspnet:6.0 #5 sha256:50f1ddc10932c4a74c7af5704e931a9489c710faea4f2381fe2380827a900e00 #5 DONE 0.0s  #8 [build 1/7] FROM mcr.microsoft.com/dotnet/sdk:6.0 #8 sha256:7ecb41e81f15ad01ec9dfe74e314966dca2b7e75c8209ba0f5971e2896aeacb1 #8 DONE 0.0s  #6 [base 2/2] WORKDIR /app #6 sha256:bc3cf6c390e5fe0e66017b4845c8fcf3e56c7adac9f514a00c2c986024f377a4 #6 CACHED  #7 [final 1/2] WORKDIR /app #7 sha256:3baba3edbb555769ad9a49e827de88d7dcc4f72ad6e2b42e576964e4a30eac96 #7 CACHED  #9 [build 2/7] WORKDIR /src #9 sha256:5ab03db9c756d494d26f6f951935e98641cd0ac874530e7c026c9e03c0da1894 #9 CACHED  #10 [internal] load build context #10 sha256:072400af91cf9df636bd0beaed4e99f3e46ac38fe86147eea7b48e38d1977cc0 #10 transferring context: 18.68kB 0.0s done #10 DONE 0.1s  #11 [build 3/7] COPY [JWTAuth_Validation/JWTAuth_Validation.csproj, JWTAuth_Validation/] #11 sha256:fd781e052dfc863ea065630ac7dc30cedea14ea90b6e10c73e96730c1e57286f #11 DONE 0.1s  #12 [build 4/7] RUN dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\" #12 sha256:e72628f0f203bca1c52d62c01c154c1b4871ce3a4f3975e9ffa4a9b518bf0054 #12 2.642   Determining projects to restore... #12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error : Unable to load the service index for source https://api.nuget.org/v3/index.json. [/src/JWTAuth_Validation/JWTAuth_Validation.csproj] #12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error :   The SSL connection could not be established, see inner exception. [/src/JWTAuth_Validation/JWTAuth_Validation.csproj] #12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error :   The remote certificate is invalid because of errors in the certificate chain: PartialChain [/src/JWTAuth_Validation/JWTAuth_Validation.csproj] #12 ERROR: executor failed running [/bin/sh -c dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\"]: exit code: 1 ------  > [build 4/7] RUN dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\": ------ executor failed running [/bin/sh -c dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\"]: exit code: 1","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__180.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__26.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"Dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"#1 [internal] load build definition from Dockerfile\r\n#1 sha256:9313b2f44313230c943697ab74e683401541514febcc03e36b4553bae24241de\r\n#1 transferring dockerfile: 32B 0.0s done\r\n#1 DONE 0.0s\r\n\r\n#2 [internal] load .dockerignore\r\n#2 sha256:037160032069b1e7222fef32a9e08ed96a2b15a56ff8e18c59ced39e47170e48\r\n#2 transferring context: 35B done\r\n#2 DONE 0.0s\r\n\r\n#3 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0\r\n#3 sha256:9eb4f6c3944cfcbfe18b9f1a753c769fc35341309a8d4a21f8937f47e94c712b\r\n#3 DONE 0.0s\r\n\r\n#4 [internal] load metadata for mcr.microsoft.com/dotnet/aspnet:6.0\r\n#4 sha256:ac4494cbca04ddb415c76edcbcc7688784c2a6ea65dd656286c013738aa3b75f\r\n#4 DONE 0.0s\r\n\r\n#5 [base 1/2] FROM mcr.microsoft.com/dotnet/aspnet:6.0\r\n#5 sha256:50f1ddc10932c4a74c7af5704e931a9489c710faea4f2381fe2380827a900e00\r\n#5 DONE 0.0s\r\n\r\n#8 [build 1/7] FROM mcr.microsoft.com/dotnet/sdk:6.0\r\n#8 sha256:7ecb41e81f15ad01ec9dfe74e314966dca2b7e75c8209ba0f5971e2896aeacb1\r\n#8 DONE 0.0s\r\n\r\n#6 [base 2/2] WORKDIR /app\r\n#6 sha256:bc3cf6c390e5fe0e66017b4845c8fcf3e56c7adac9f514a00c2c986024f377a4\r\n#6 CACHED\r\n\r\n#7 [final 1/2] WORKDIR /app\r\n#7 sha256:3baba3edbb555769ad9a49e827de88d7dcc4f72ad6e2b42e576964e4a30eac96\r\n#7 CACHED\r\n\r\n#9 [build 2/7] WORKDIR /src\r\n#9 sha256:5ab03db9c756d494d26f6f951935e98641cd0ac874530e7c026c9e03c0da1894\r\n#9 CACHED\r\n\r\n#10 [internal] load build context\r\n#10 sha256:072400af91cf9df636bd0beaed4e99f3e46ac38fe86147eea7b48e38d1977cc0\r\n#10 transferring context: 18.68kB 0.0s done\r\n#10 DONE 0.1s\r\n\r\n#11 [build 3/7] COPY [JWTAuth_Validation/JWTAuth_Validation.csproj, JWTAuth_Validation/]\r\n#11 sha256:fd781e052dfc863ea065630ac7dc30cedea14ea90b6e10c73e96730c1e57286f\r\n#11 DONE 0.1s\r\n\r\n#12 [build 4/7] RUN dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\"\r\n#12 sha256:e72628f0f203bca1c52d62c01c154c1b4871ce3a4f3975e9ffa4a9b518bf0054\r\n#12 2.642   Determining projects to restore...\r\n#12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error : Unable to load the service index for source https://api.nuget.org/v3/index.json. [/src/JWTAuth_Validation/JWTAuth_Validation.csproj]\r\n#12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error :   The SSL connection could not be established, see inner exception. [/src/JWTAuth_Validation/JWTAuth_Validation.csproj]\r\n#12 6.180 /usr/share/dotnet/sdk/6.0.101/NuGet.targets(130,5): error :   The remote certificate is invalid because of errors in the certificate chain: PartialChain [/src/JWTAuth_Validation/JWTAuth_Validation.csproj]\r\n#12 ERROR: executor failed running [/bin/sh -c dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\"]: exit code: 1\r\n------\r\n > [build 4/7] RUN dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\":\r\n------\r\nexecutor failed running [/bin/sh -c dotnet restore \"JWTAuth_Validation/JWTAuth_Validation.csproj\"]: exit code: 1"}]